.azure/OneBranch.PullRequest.yml

```diff
@@ -201,6 +201,12 @@ extends:
       - build_linux
       jobs:
       - template: .azure/obtemplates/build-linux-packages.yml@self
```

Comment:
Let's be explicit about the `kernel` parameter being passed in here, so the reader doesn't need to go look at the template to figure out the default. Same for Official yml.